scripts/checks

```diff
@@ -11,20 +11,51 @@ fi
 
 BUILD_PKGS=$1
 
-echo "Checking Go version..."
+echo "+++ Checking Go version..."
 ver=$(go version | awk '{print $3}')
 minVer="go1.4"
 if [[ ${ver} < ${minVer} ]]; then
-    echo "Go version check failed. Expected >=${minVer} but found ${ver}"
+    echo "!!! Go version check failed. Expected >=${minVer} but found ${ver}"
     exit 1
 fi
 
-echo "Checking gofmt..."
+echo "+++ Checking gofmt..."
 fmtRes=$(gofmt -l $BUILD_PKGS)
 if [ -n "${fmtRes}" ]; then
-    echo -e "gofmt checking failed:\n${fmtRes}"
+    echo -e "!!! gofmt checking failed:\n${fmtRes}"
     exit 1
 fi
 
-echo "All checks passed!!"
+if ! go get github.com/golang/lint/golint
+then
+  echo "!!! Could not install golint"
+  exit 1
+fi
+
+echo "+++ golint tree..."
+for i in $(find . -maxdepth 1 -type d -name '*' | grep -vE '/\..*$')
```

Comment:
The `checks` script takes `BUILD_PKGS` as argument that contains the list of our modules. We can alternatively iterate over those and then perhaps not need the `find`. It will keep this check consistent with rest of the checks as well.

```
lintRes=$(for dir in $BUILD_PKGS; do golint $dir; done)
if [ -n "${lintRes}" ]; then
    echo -e "!!! golint has suggestions:\n${lintRes}"
fi
```